specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/informationProtection.json

```diff
@@ -0,0 +1,310 @@
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "InformationProtectionPolicy": {
+            "type": "object",
+            "description": "Information protection policy",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "description": "Information protection policy data",
+                    "$ref": "#/definitions/InformationProtectionPolicyProperties"
+                }
+            },
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                }
+            ]
+        },
+        "InformationProtectionPolicyProperties": {
+            "type": "object",
+            "description": "describes properties of an information protection policy",
+            "properties": {
+                "lastModifiedUtc": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "Describes the last UTC time the policy was modified"
+                },
+                "sensitivityLabels": {
+                    "type": "array",
+                    "description": "List of sensitivity labels",
+                    "items": {
+                        "$ref": "#/definitions/SensitivityLabel"
+                    }
+                },
+                "sensitivityInformationTypes": {
+                    "type": "array",
+                    "description": "List of sensitivity information types",
+                    "items": {
+                        "$ref": "#/definitions/SensitivityInformationType"
+                    }
+                }
+            }
+        },
+        "SensitivityLabel": {
+            "type": "object",
+            "description": "The sensitivity label.",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "The id of the sensitivty label."
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the sensitivity label."
+                },
+                "order": {
+                    "type": "number",
+                    "description": "The order of the sensitivity label."
+                }
+            }
+        },
+        "SensitivityInformationType": {
+            "type": "object",
+            "description": "The information type.",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "The id of the information type."
+                },
+                "name": {
+                    "type": "string",
+                    "description": "The name of the information type."
+                },
+                "order": {
```

Comment:
Does order have a limit? Why does order need to exist if the data type is within an array?

Reply:
We changed it to a dictionary